homeassistant/components/alarm_control_panel/device_trigger.py

```diff
@@ -143,13 +142,10 @@ async def async_attach_trigger(
         from_state = STATE_ALARM_DISARMED
         to_state = STATE_ALARM_ARMING
     elif config[CONF_TYPE] == "armed_home":
-        from_state = STATE_ALARM_PENDING or STATE_ALARM_ARMING
```

Comment:
We had a from_state (although this is obviously wrong) because otherwise if an attribute updates, the state remains the same and it shouldn't trigger.

Reply:
I don't think that's correct: https://github.com/home-assistant/core/blob/dev/homeassistant/components/homeassistant/triggers/state.py#L109-L121

Reply:
We only trigger on attributes if *both* `from_state` and `to_state` are set to `*` (or not specified)

Reply:
You are right and I am wrong.